parm/fv3lam_post_avblflds.xml

```diff
@@ -5678,6 +5678,41 @@
        <scale>4.0</scale>
     </param>
 
+    <param>
```

Comment:
@SamuelTrahanNOAA The UPP avblflds xml files were unified as post_avblflds.xml. We will remove 
 this legacy fv3lam_post_avblflds.xml soon. So your new changes are not needed in this file.

Reply:
The fields are in post_avblflds.xml too.

Reply:
@SamuelTrahanNOAA Somehow, I don't see these changes in post_avblflds.xml.